packages/framework/src/Model/Cart/CartFacade.php

```diff
@@ -252,6 +251,9 @@ public function findCartByCustomerUserIdentifier(CustomerUserIdentifier $custome
 
                 return null;
             }
+
+            $cart->setModifiedNow();
+            $this->em->flush();
```

Comment:
I am wondering if this is necessary for loading cart. I think that changing modified just during update of cart would be sufficient. Same for `getCartByCustomerUserIdentifierCreateIfNotExists`

Reply:
in acceptance criterie is "since the user's last activity" , this i uderstand as visit any page and not just add/remove item or clean cart.